app/models/microcosm.rb

```diff
@@ -0,0 +1,43 @@
+# == Schema Information
+#
+# Table name: microcosms
+#
+#  id          :bigint(8)        not null, primary key
+#  name        :string           not null
+#  description :text
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  slug        :string           not null
+#  location    :string           not null
+#  lat         :decimal(, )      not null
+#  lon         :decimal(, )      not null
```

Comment:
We normally use either integers (e.g. for nodes) or floats (e.g. for user home locations).

Reply:
Yeah, you can see below in this file the mins and maxes are integers.  Not sure how these two went rogue.  Fixed.